ting.admin.inc

```diff
@@ -35,22 +35,6 @@ function ting_admin_ting_settings($form_state) {
     '#default_value' => variable_get('ting_search_url', ''),
   );
 
-  $form['ting']['ting_scan_url'] = array(
-    '#type' => 'textfield',
-    '#title' => t('Scan service URL'),
-    '#description' => t('URL to the Ting scan webservice, e.g. http://openscan.addi.dk/2.0/'),
-    '#required' => TRUE,
-    '#default_value' => variable_get('ting_scan_url', ''),
-  );
-
-  $form['ting']['ting_spell_url'] = array(
-    '#type' => 'textfield',
-    '#title' => t('Spell service URL'),
-    '#description' => t('URL to the Ting spell webservice, e.g. http://openspell.addi.dk/1.2/'),
-    '#required' => TRUE,
-    '#default_value' => variable_get('ting_spell_url', ''),
-  );
-
```

Comment:
This removes the form elements for the preceding services but I do not see any new ones being added. I guess the are now moved to `ting_search_autocomplete` and I am OK with that but it needs to be clear when reading the code.